server/service/src/sync/program_schemas/hiv_testing_encounter_ui_schema.json

```diff
@@ -346,56 +353,69 @@
           "scope": "#/properties/hivTesting/properties/postTestCounsellingDone",
           "label": "Post-test counselling done?"
         },
-        { "type": "Spacer" },
         {
           "type": "Control",
-          "scope": "#/properties/hivTesting/properties/referredForARTServices",
-          "label": "Did you refer the patient for ART services?"
+          "scope": "#/properties/hivTesting/properties/serodiscordantCoupleCounsellingProvided",
+          "label": "If serodiscordant, was couple counselling provided?"
         },
         {
-          "type": "Control",
-          "scope": "#/properties/hivTesting/properties/referredARTFacility",
-          "label": "Where?",
-          "options": {
-            "show": [
-              ["THIS_FACILITY", "This facility"],
-              ["OTHER_FACILITY", "Other facility"]
-            ]
-          },
-          "rule": {
-            "effect": "ENABLE",
-            "condition": {
+          "type": "Group",
+          "elements": [
+            { "type": "Spacer" },
+            {
+              "type": "Control",
               "scope": "#/properties/hivTesting/properties/referredForARTServices",
-              "schema": {
-                "const": true
+              "label": "Did you refer the patient for ART services?"
+            },
+            {
+              "type": "Control",
+              "scope": "#/properties/hivTesting/properties/referredARTFacility",
+              "label": "Where?",
+              "options": {
+                "show": [
+                  ["THIS_FACILITY", "This facility"],
+                  ["OTHER_FACILITY", "Other facility"]
+                ]
+              },
+              "rule": {
+                "effect": "ENABLE",
+                "condition": {
+                  "scope": "#/properties/hivTesting/properties/referredForARTServices",
+                  "schema": {
+                    "const": true
+                  }
+                }
               }
+            },
+            {
+              "type": "Control",
+              "scope": "#/properties/hivTesting/properties/referredARTFacilityName",
+              "label": "If another facility, name of the facility",
+              "rule": {
+                "effect": "ENABLE",
+                "condition": {
+                  "scope": "#/properties/hivTesting/properties/referredARTFacility",
+                  "schema": { "enum": ["OTHER_FACILITY"] }
+                }
+              }
+            },
+            {
+              "type": "Control",
+              "scope": "#/properties/hivTesting/properties/clientEscorted",
+              "label": "Was the client escorted?"
             }
-          }
-        },
-        {
-          "type": "Control",
-          "scope": "#/properties/hivTesting/properties/referredARTFacilityName",
-          "label": "If another facility, name of the facility",
+          ],
           "rule": {
-            "effect": "ENABLE",
```

Comment:
Hide the whole section if the result isn't positive since the fields seem to be dependant on each other

Reply:
@richardmoizeau at least for me this show/hide is very difficult to see in the UI, e.g. for me its unclear which section became visible/invisible. Maybe move this "ART service" section one section up so that the hidden group is closer to the control that triggers the hidding?